src/jit/compiler.cpp

```diff
@@ -5737,6 +5737,9 @@ var_types Compiler::GetTypeFromClassificationAndSizes(SystemVClassificationType
     case SystemVClassificationTypeIntegerReference:
         type = TYP_REF;
         break;
+    case SystemVClassificationTypeIntegerByReference:
```

Comment:
This is utility method used by other code to get the Jit type off classificationType and size of the non padded eightbyte.